### ISSUE TITLE
Can't open new on-chain proposal page

### ISSUE THREAD

**Issue by masvelio** [hicommonwealth/commonwealth#3258]
Tried with multiple communities locally, but always got infinite cow.

![Image](https://user-images.githubusercontent.com/14819225/227940769-aaa5d0e3-f978-47d3-a792-3083b60e9f99.png)

It sometimes works on remote react env though,  but also got infinite cow multiple times https://commonwealth-react.herokuapp.com/dydx/new/proposal

![Image](https://user-images.githubusercontent.com/14819225/227942312-f8a05977-b7f9-45cc-abbf-b73babcf7896.png)

**Comment by jnaviask** [comment]
Fixed by #3231 ?

**Comment by CowMuon** [comment]
@martins87 Can you confirm whether this is still an issue per Jake's comment?

**Comment by CowMuon** [comment]
@zakhap This is not critical path?

**Comment by martins87** [comment]
> @martins87 Can you confirm whether this is still an issue per Jake's comment?

Did you mean @masvelio ?

**Comment by dillchen** [comment]
It likely should be critical path

**Comment by masvelio** [comment]
@jnaviask it does not work for me on local env, but seems to work fine on remote env. Am I missing env variables too? Can you get me those which are necessary to run this locally?

**Comment by jnaviask** [comment]
@masvelio which communities did you try? I was able to get dydx open locally. But looks like this may be blocked on #3335.

**Comment by jnaviask** [comment]
Only occurs when you navigate directly to the new proposal page. Still outstanding though.

**Comment by dillchen** [comment]
@jnaviask can we move this out of backlog? Should be resolved this week during "QA", as it's a regression

**Comment by dillchen** [comment]
If push comes to shove, should be right after react deploy

**Comment by masvelio** [comment]
@jnaviask I tried dydx and locally I cant open it. On remote env seems to work fine

![Image](https://user-images.githubusercontent.com/14819225/231132886-501ab8af-c9c3-424f-a0b1-f8296082f228.png)

**Comment by masvelio** [comment]
@jnaviask this is still reproducible for me both in https://commonwealth-beta.herokuapp.com/dydx/new/proposal and https://commonwealth-react.herokuapp.com/dydx/new/proposal

**Comment by mzparacha** [comment]
PR https://github.com/hicommonwealth/commonwealth/pull/3476 created for this issue. Also found another bug while working on this, fixed it in the same PR.

**Comment by jnaviask** [comment]
Fix in #3476 caused regression in thread loading. Reopening.

**Comment by mzparacha** [comment]
Related to https://github.com/hicommonwealth/commonwealth/issues/3598

**Comment by jnaviask** [comment]
May have been fixed by #3673. Will create new ticket if outstanding.
